src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,224 @@
+import {
+  getVAAppointmentLocationId,
+  getVARFacilityId,
+  isVideoAppointment,
+} from '../../services/appointment';
+import { FETCH_STATUS, PURPOSE_TEXT } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import * as actions from '../redux/actions';
+import { cancelAppointment } from '../redux/actions';
+import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
+import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
+
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
+  return id ? id.replace('var', '') : id;
+}
+
+function formatAppointmentDate(date) {
+  if (!date.isValid()) {
+    return null;
+  }
+
+  return date.format('MMMM D, YYYY');
+}
+
+function ConfirmedAppointmentDetailsPage({
+  appointmentDetails,
+  appointmentDetailsStatus,
+  facilityData,
+  fetchConfirmedAppointmentDetails,
+  confirmedStatus,
+  showCancelButton,
+}) {
+  const { id } = useParams();
+  const history = useHistory();
+
+  const appointment = appointmentDetails?.[id];
+
+  useEffect(() => {
+    const status = confirmedStatus === FETCH_STATUS.succeeded;
+
+    if (!status) {
+      history.push('/va');
+    }
+
+    if (!appointment) {
+      fetchConfirmedAppointmentDetails(id);
+    }
+
+    scrollAndFocus();
+  }, []);
+
+  if (appointmentDetailsStatus === FETCH_STATUS.loading) {
+    return (
+      <div className="vads-u-margin-y--8">
+        <LoadingIndicator message="Loading your appointment request...!!!!" />
+      </div>
+    );
+  }
+
+  if (!appointment) {
+    return null;
+  }
+
+  const facilityId = getVAAppointmentLocationId(appointment);
+  const facility = facilityData?.[facilityId];
+
+  const isCC = appointment?.vaos.isCommunityCare;
+  const isExpressCare = appointment?.vaos.isExpressCare;
+  const isCommunityCare = appointment?.vaos.isCommunityCare;
+  const isVideo = isVideoAppointment(appointment);
+  const isInPersonVAAppointment = !isVideo && !isCommunityCare;
+
+  const header = 'VA Appointment';
+  const showInstructions =
+    isCommunityCare ||
+    (isInPersonVAAppointment &&
+      PURPOSE_TEXT.some(purpose =>
+        appointment?.comment?.startsWith(purpose.short),
+      ));
+
+  return (
+    <div>
+      <div className="vads-u-display--block vads-u-padding-y--2p5">
+        ‹ <Link to="/">Manage appointments</Link>
+      </div>
+
+      <h1>
+        <AppointmentDateTime
+          appointmentDate={moment.parseZone(appointment.start)}
+          timezone={appointment.vaos.timeZone}
+          facilityId={getVARFacilityId(appointment)}
+        />
+      </h1>
+
+      {!!facility &&
+        !isCC &&
+        !isExpressCare && (
+          <>
+            <div className="vads-u-font-size--sm vads-u-font-weight--normal vads-u-font-family--sans">
+              <span>
+                <b>{header}</b>
```

Comment:
Can we use `<strong>` instead of `<b>` here? Also, is there a reason we're combining vads-u-font-weight--normal with a bolding element? Looks like we're trying mimic a heading here. Should we be using a heading?

Reply:
Thanks for the review. I changed <b> to <strong> and removed the font weight style. I will check with are UI/UX person about whether his intent was for a header or bold text.

Reply:
It was decided to leave as is.